packages/edit-site/src/components/editor/global-styles-provider.js

```diff
@@ -131,7 +187,11 @@ export default function GlobalStylesProvider( {
 			styles: [
 				...newStyles,
 				{
-					css: getGlobalStyles( contexts, mergedStyles ),
+					css: getGlobalStyles(
+						contexts,
+						mergedStyles,
+						STYLE_PROPERTY
```

Comment:
Can the function. simply use the STYLE_PROPERTY constant instead of us passing it explicitly?

Reply:
I thought I'd pass it so the rendered is agnostic where this data comes from ― if we want to change it later it's less places to update.